### PR TITLE
removed carousel and compressed stock photo

### PR DESCRIPTION
**Pull request by seanrayment**
Small pull request to finally compress the stock photo on the landing page.

### CODE REVIEW

File: homePage/templates/homePage/index.html [hunk 1]
```diff
@@ -38,32 +38,34 @@ <h1>Welcome to Cocoon!</h1>
...
+      <img class="landingImage" src="{% static 'stockHomes/landing_image.jpeg' %}" alt>
+
+{#        <!-- Wrapper for slides -->#}
+{#        <div class="carousel-inner" role="listbox">#}
+{#            <div class="item active">#}
+{#                <img src="{% static 'stockHomes/carousal1.jpeg' %}" alt="...">#}
+{#                <div class="carousel-caption">#}
+{##}
+{#                </div>#}
+{#            </div>#}
```
Comment (tiusty) [inline homePage/templates/homePage/index.html]: Is this commented out?

Comment (tiusty) [reply homePage/templates/homePage/index.html]: if so we can just remove it?

**Comment by seanrayment** [comment]
That code should be removed now!

File: .gitignore [hunk 2]
```diff
@@ -10,8 +10,11 @@ houseDatabase/migrations
 homePage/migrations
 survey/migrations
 
+.vscode/
```
Comment (tiusty) [inline .gitignore]: can you comment what this is for?

**Review by tiusty — DISMISSED** [review]

**Pull request closed without merging**